pkg/localstore/mode_put.go

```diff
@@ -85,13 +85,6 @@ func (db *DB) put(mode storage.ModePut, chs ...swarm.Chunk) (exist []bool, err e
 			}
 			exist[i] = exists
 			gcSizeChange += c
-
-			if mode == storage.ModePutRequestPin {
```

Comment:
I'm reinstating this part since its needed, but I'm not sure how this will behave with the `gcSizeChange` line at 87